### ISSUE TITLE
Recent WebGLRenderTargetCube bug?

### ISSUE THREAD

**Issue by MKHenson** [mrdoob/three.js#5985]
Hi guys

I am using three for a small project and was implementing the WebGLRenderTargetCube for a scene. This was working fine up until last week, but now seems to be broken (probably from recent browser updates). On chrome there is no error, but the render targets are black; while in Firefox there is an error and reads as:

"Error: WebGL: A texture is going to be rendered as if it were black, as per the OpenGL ES 2.0.24 spec section 3.8.2, because it is a cube map texture, with a minification filter requiring a mipmap, and is not mipmap cube complete (as defined in section 3.7.10)."

Checked the examples and its happening here for me:
http://threejs.org/examples/#webgl_materials_cubemap_dynamic2

**Comment by mrdoob** [comment]
You need to tell us what OS and version you're using, what graphics card, does it happen in other computers, etc

**Comment by MKHenson** [comment]
Hi mrdoob. The firefox version is 35.0.1, OS is Windows 7, and graphics card is NVidia GeForce GTX 680

![error](https://cloud.githubusercontent.com/assets/2727104/5910540/12dc94e2-a5b3-11e4-9533-7b260bad346d.png)

**Comment by MKHenson** [comment]
I don't have another computer at the moment, but if I can get my hands on one I will let you know

**Comment by mrdoob** [comment]
Oh! Windows... Are you running the latest drivers for that graphics card?

**Comment by MKHenson** [comment]
Unfortunately updating the drivers did nothing, the problem still seems to exist :/ I tried something in my app which seems to have fixed my problem, though Im not sure why it would have to be done. If I setup the min texture filter to be NEAREST_MIPMAP_LINEAR, then the problem seems to go away. Before that I had tried both Linear and Nearest with no mipmapping enabled and FF would blacken the shader and give me the error. Does it mean that we have to use mipmaping with cube textures? (I read the spec they reference and it doesnt seem that way to me...)
